arkouda/pdarraysetops.py

```diff
@@ -414,10 +418,10 @@ def union1d(
 
```

Comment:
most of the functions in this file have a `See Also` to `unique`. If we want that to link to the function, i think we gotta do `groupbyclass.unique` or something similar

Reply:
updated